lib/wallaroo/core/autoscale/autoscale_phase.pony

```diff
@@ -692,11 +730,11 @@ class _WaitingForBoundariesMap is _AutoscalePhase
 
   fun name(): String => "_WaitingForBoundariesMap"
 
-  fun ref worker_join(conn: TCPConnection, worker: WorkerName,
-    worker_count: USize, local_topology: LocalTopology,
-    current_worker_count: USize)
+  fun ref try_join(local_topology: LocalTopologyInitializer,
```

Comment:
I don't think this should be implemented here, since we are in-progress for an autoscale event.